### PR TITLE
Feature shooter patch

### PR DESCRIPTION
**Pull request by JettK**
Deleted ball trajectory

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -22,6 +24,8 @@
 
   private RobotContainer m_robotContainer;
 
+  private BallTrajectory m_balltrajectory;
```
Comment (bartkerfeld) [inline src/main/java/frc/robot/Robot.java]: Remove this

File: src/main/java/frc/robot/Robot.java [hunk 2]
```diff
@@ -101,6 +105,7 @@ public void teleopInit() {
    */
   @Override
   public void teleopPeriodic() {
+    // SmartDashboard.putNumber("Ball Exit Trajectory", m_balltrajectory.getExitVelocity());
```
Comment (bartkerfeld) [inline src/main/java/frc/robot/Robot.java:105]: Remove this

File: src/main/java/frc/robot/RobotContainer.java [hunk 3]
```diff
@@ -35,15 +38,18 @@
 public class RobotContainer {
   // The robot's subsystems and commands are defined here...
   private final FlywheelSubsystem flywheelsubsystem = new FlywheelSubsystem();
+  private final ShooterHoodSubsystem shooterhoodsubsystem = new ShooterHoodSubsystem();
```
Comment (bartkerfeld) [inline src/main/java/frc/robot/RobotContainer.java]: rename to m_hood or something with the 'm_' prefix

File: src/main/java/frc/robot/subsystems/shooter/ShooterHoodSubsystem.java [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.RunCommand;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants.ConversionConstants;
+import frc.robot.Constants.PIDConstants;
+import frc.robot.Constants.RobotMap;
+/**
+   * Creates a new ShooterHoodSubsystem.
+   */
+public class ShooterHoodSubsystem extends SubsystemBase {
```
Comment (bartkerfeld) [inline src/main/java/frc/robot/subsystems/shooter/ShooterHoodSubsystem.java]: Do we need the word Shooter in the name? I think HoodSubsystem is just as clear and is shorter.

File: src/main/java/frc/robot/subsystems/shooter/ShooterHoodSubsystem.java [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+    m_hoodMotor.config_kI(0, SmartDashboard.getNumber("Hood kI", 0));
+    m_hoodMotor.config_IntegralZone(0, (int)SmartDashboard.getNumber("Hood IntegralZone", 0));
+    m_hoodMotor.config_kD(0, newkD);
+    m_hoodMotor.config_kF(0, newkF);
+
+    System.out.println("Hood configed");
+  }
+
+  public void setHoodDegrees(){
+    double target = degreesToSRX(SmartDashboard.getNumber("Hood Target Degrees", 0));
```
Comment (bartkerfeld) [inline src/main/java/frc/robot/subsystems/shooter/ShooterHoodSubsystem.java]: This is fine for now, but we should find an alternative method to pass this information around that doesn't require writing and reading from NetworkTables. It adds unnecessary network delays.

Comment (JettK) [reply src/main/java/frc/robot/subsystems/shooter/ShooterHoodSubsystem.java]: Once testing is done we'll set gains in the constructory

File: src/main/java/frc/robot/subsystems/shooter/ShooterHoodSubsystem.java (hunk 4 above)
Comment (JettK) [reply src/main/java/frc/robot/subsystems/shooter/ShooterHoodSubsystem.java]: Sure

File: src/main/java/frc/robot/RobotContainer.java (hunk 3 above)
Comment (JettK) [reply src/main/java/frc/robot/RobotContainer.java]: Done

File: src/main/java/frc/robot/subsystems/shooter/ShooterHoodSubsystem.java (hunk 5 above)
Comment (bartkerfeld) [reply src/main/java/frc/robot/subsystems/shooter/ShooterHoodSubsystem.java]: It is not the gains I am talking about. I am talking about the target hood position.

File: src/main/java/frc/robot/Constants.java [hunk 6]
```diff
@@ -33,12 +26,12 @@
 public final class Constants {
     // Control Panel
     public static final double kControlPanelCircumference = 0.81 * Math.PI;
-
     public static final double kColorArcLength = kControlPanelCircumference / 8;
-    
     public static final double kManipulatorCircumference = 0.0508 * Math.PI;
-    
-    public static double threeTurns = 26.0; // Rotate 26 segments 
+    public static final double threeTurns = 26.0; // Rotate 26 segments
```
Comment (bartkerfeld) [inline src/main/java/frc/robot/Constants.java:31]: Why was this added in this branch? This seems control panel specific, not shooter specific
